### ISSUE TITLE
Viewing others' plans as non-admin

### ISSUE THREAD

**Issue by gasnew** [klavinslab/aquarium-bugs#33]
This would be nice for collaborating (particularly with someone working on the same project or with an undergrad assistant).

Though, would we want limited permissions for this? Maybe a user can specify which other users can view their plans in their account preferences?

**Comment by klavins** [comment]
Yeah, we could use groups. People also want to limit who can see their samples, so that could serve the same purpose.
